tftpsync/susemanager-tftpsync-recv/add.wsgi

```diff
@@ -45,17 +43,19 @@ ch.setFormatter(formatter)
 # add ch to logger
 logger.addHandler(ch)
 
+
 class TftpFieldStorage(cgi.FieldStorage):
 
     def make_file(self, binary=None):
         tmpdir = os.path.join(CFG.TFTPBOOT, "tmp")
         if not os.path.exists(tmpdir):
-                os.makedirs(tmpdir)
+            os.makedirs(tmpdir)
```

Comment:
For me it looks like the race condition could appear here also.
Maybe avoid it this way:
```suggestion
            os.makedirs(tmpdir, exist_ok=True)
```